packages/postcss-colormin/src/index.js

```diff
@@ -66,24 +66,29 @@ function transform(value, options) {
   return parsed.toString();
 }
 
-function pluginCreator() {
+function addPluginDefaults(options, browsers) {
+  const defaults = {
+    transparent: browsers.some(hasTransparentBug) === false, // Does the browser support 4 & 8 character hex notation
```

Comment:
Nice to check for feature support!

Reply:
This was already in the code, I just moved it around. 🙈

Reply:
Ah, true, but it still a bugfix because previously the options did not get passed down to colord.